v2.0/install-cockroachdb.html

```diff
@@ -8,67 +8,32 @@
 
 <script>
 $(document).ready(function(){
-
-    //detect os and display corresponding tab by default
-    if (navigator.appVersion.indexOf("Mac")!=-1) {
-        $('#os-tabs').find('button').removeClass('current');
-        $('#mac').addClass('current');
-        toggleMac();
-    }
-    if (navigator.appVersion.indexOf("Linux")!=-1) {
-        $('#os-tabs').find('button').removeClass('current');
-        $('#linux').addClass('current');
-        toggleLinux();
-    }
-    if (navigator.appVersion.indexOf("Win")!=-1) {
-        $('#os-tabs').find('button').removeClass('current');
-        $('#windows').addClass('current');
-        toggleWindows();
-    }
-
-    var install_option = $('.install-option'),
-        install_button = $('.install-button');
-
-    install_button.on('click', function(e){
-      e.preventDefault();
-      var hash = $(this).prop("hash");
-
-      install_button.removeClass('current');
-      $(this).addClass('current');
-      install_option.hide();
-      $(hash).show();
-
+    $('.install-button').on('click', function(e) {
+        e.preventDefault();
+        $('.install-button').removeClass('current');
+        $(this).addClass('current');
+        $('.install-option').hide();
+        $($(this).prop("hash")).show();
     });
 
     //handle click event for os-tab buttons
     $('#os-tabs').on('click', 'button', function(){
         $('#os-tabs').find('button').removeClass('current');
         $(this).addClass('current');
 
-        if($(this).is('#mac')){ toggleMac(); }
-        if($(this).is('#linux')){ toggleLinux(); }
-        if($(this).is('#windows')){ toggleWindows(); }
+        $("#macinstall,#linuxinstall,#windowsinstall").hide();
+        var os = $(this).attr("id");
+        $("#" + os + "install").show();
+        $("." + os + "-button:first").click();
     });
 
-    function toggleMac(){
-        $(".mac-button:first").trigger('click');
-        $("#macinstall").show();
-        $("#linuxinstall").hide();
-        $("#windowsinstall").hide();
-    }
-
-    function toggleLinux(){
-        $(".linux-button:first").trigger('click');
-        $("#linuxinstall").show();
-        $("#macinstall").hide();
-        $("#windowsinstall").hide();
-    }
-
-    function toggleWindows(){
-        $(".windows-button:first").trigger('click');
-        $("#windowsinstall").show();
-        $("#macinstall").hide();
-        $("#linuxinstall").hide();
+    // Detect OS and display corresponding tab by default.
```

Comment:
@m-schneider, who's on linux, said this defaults to linux in chrome but to mac in ff. Any ideas why?

Also, don't you want to make these changes to the 1.1 version of the page as well?

Reply:
Hmm, as a result of this change, or always?